app/models/concerns/reviewable_action_builder.rb

```diff
@@ -129,17 +125,38 @@ def build_new_separated_actions(actions, guardian, args)
     raise NotImplementedError, "Including class must implement #build_new_separated_actions"
   end
 
+  # Build a bundle of actions and add it to the provided actions list.
+  #
+  # @param actions [Reviewable::Actions] Actions instance to add to.
+  # @param id [String] ID for the bundle, used to derive I18n keys.
+  # @param label [String] I18n key for the bundle label.
+  # @param bundle_actions [Hash] Hash of action IDs and optional params to pass to build_action.
+  # @option bundle_actions [Symbol] :client_action Optional client-side action identifier (e.g. "edit").
+  # @option bundle_actions [Symbol] :confirm When true, uses "reviewables.actions.<id>.confirm" for confirm_message.
+  # @option bundle_actions [Symbol] :require_reject_reason When true, requires a rejection reason for the action.
+  # @param source [String] Optional source string for namespacing I18n keys. Will default to `type_source`.
+  #
+  # @return [Reviewable::Actions::Bundle] The created bundle.
+  def build_bundle(actions, id, label, bundle_actions = {}, source: nil)
+    bundle = actions.add_bundle(id, label:)
+    bundle_actions.each do |action_id, action_params|
+      build_action(actions, action_id, bundle:, **action_params || {}, source:)
+    end
+    bundle
+  end
```

Comment:
@lis2, @Drenmi: I'm interested in your thoughts on this helper for creating action bundles. I think it does a reasonable job of simplifying writing custom bundles, particularly with no longer needing to constantly pass around the `actions` and `bundle` blobs.

I've updated `ReviewableActionBuilder#build_post_actions_bundle`, `ReviewableActionBuilder#build_user_actions_bundle`, and `Chat::ReviewableMessage#build_new_separated_actions` to use this helper, but I'll extract it out into a separate PR if we think it has merit.